src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,118 @@
+                        <FormattedMessage {...messages.versionsTitle} />
+                    </>
+                </h3>
+            </div>
+            <div className="bcs-scroll-content-wrapper">
+                <div className="bcs-scroll-content">
+                    <LoadingIndicatorWrapper
+                        className="bcs-Versions-content"
+                        crawlerPosition="top"
+                        isLoading={isLoading}
+                    >
+                        <div className="bcs-Versions-menu">
+                            <ul className="bcs-VersionsMenu">
+                                <li className="bcs-VersionsMenu-item" key="versionsToday">
+                                    <section className="bcs-VersionsGroup">
+                                        <h4 className="bcs-VersionsGroup-heading">
+                                            <FormattedMessage {...messages.versionsToday} />
+                                        </h4>
+                                        <ul className="bcs-VersionsList">
+                                            <li className="bcs-VersionsList-item" key={1}>
+                                                <StaticVersionsItem
+                                                    versionNumber="1"
+                                                    versionTime="2021-11-29T15:01:19-08:00"
+                                                />
+                                            </li>
+                                            <li className="bcs-VersionsList-item" key={2}>
+                                                <StaticVersionsItem
+                                                    versionNumber="2"
+                                                    versionTime="2021-11-29T15:01:19-08:00"
+                                                />
+                                            </li>
+                                            <li className="bcs-VersionsList-item" key={3}>
+                                                <StaticVersionsItem
+                                                    versionNumber="3"
+                                                    versionTime="2021-11-29T15:01:19-08:00"
+                                                />
+                                            </li>
+                                        </ul>
+                                    </section>
+                                </li>
+                            </ul>{' '}
+                        </div>
+                    </LoadingIndicatorWrapper>
+                </div>
+            </div>
+            <div className="bcs-upsell-wrapper">
+                <div className="bcs-version-upsell">
+                    {showUpsellWithPicture && <BoxDrive140 className="bcs-version-upsell-icon" />}
+                    <p className="bcs-upgrade-now">Upgrade Now</p>
```

Comment:
Should we put this copy into messages as well?

Reply:
yup! Moving it now